java/wavefrontsender.md

```diff
@@ -5,103 +5,147 @@
 * Batch size - Amount of data to send to Wavefront in each flush interval.
 
-Together, the batch size and flush interval control the maximum theoretical throughput of the `WavefrontSender`. Override the defaults _only_ to set higher values.
+Together, the batch size and flush interval control the maximum theoretical throughput of the `WavefrontSender`. 
+> **Note**: Override the defaults values only when you want to set higher values.
+
+**Example**: Use a factory class to create a WavefrontClient and send  data to Wavefront via direct ingestion.
+
+```java
+// Create a factory and add a client with the following URL format: "http://TOKEN@DOMAIN.wavefront.com"
+// and a Wavefront API token with direct ingestion permission
+WavefrontClientFactory wavefrontClientFactory = new WavefrontClientFactory();
+
+// Add a new client that sends data directly to Wavefront services 
+wavefrontClientFactory.addClient(wavefrontURL);
+
+WavefrontSender wavefrontSender = wavefrontClientFactory.getClient();
+```
+
+**Example**: Use a builder to create a WavefrontClient and send  data to Wavefront via direct ingestion.
 
 ```java
-// Create a builder with the Wavefront URL and a Wavefront API token
-// that was created with direct ingestion permission.
-WavefrontDirectIngestionClient.Builder wfDirectIngestionClientBuilder =
-  new WavefrontDirectIngestionClient.Builder(wavefrontURL, token);
+// Using the WavefrontClient.Builder directly with a url in the form of "https://DOMAIN.wavefront.com"
+// and a Wavefront API token with direct ingestion permission
+WavefrontClient.Builder wfClientBuilder = new WavefrontClient.Builder(wavefrontURL, token);
+
+// This is the size of internal buffer beyond which data is dropped
+// Optional: Set this to override the default max queue size of 50,000
+wfClientBuilder.maxQueueSize(100_000);
+
+// This is the max batch of data sent per flush interval
+// Optional: Set this to override the default batch size of 10,000
+wfClientBuilder.batchSize(20_000);
 
-// Optional: Override the max queue size (in data points). Default: 50,000
-wfDirectIngestionClientBuilder.maxQueueSize(100_000);
+// Together with batch size controls the max theoretical throughput of the sender
+// Optional: Set this to override the default flush interval value of 1 second
+wfClientBuilder.flushIntervalSeconds(2);
 
-// Optional: Override the batch size (in data points). Default: 10,000
-wfDirectIngestionClientBuilder.batchSize(20_000);
+WavefrontSender wavefrontSender = wfClientBuilder.build();
+``` 
 
-// Optional: Override the flush interval (in seconds). Default: 1 second
-wfDirectIngestionClientBuilder.flushIntervalSeconds(2);
+## Send Data to Multiple Wavefront Services
 
-// Create a WavefrontDirectIngestionClient.
-WavefrontSender wavefrontSender = wfDirectIngestionClientBuilder.build();
- ```
+Use `WavefrontMultiClient` to send data to multiple Wavefront services so you handle the data traffic.
```

Comment:
Do we need more detail/background info? I find this confusing. 
* Why would I have multiple Wavefront services? 
* If I do, wouldn't they sit behind a load balancer?

Reply:
thank you! This is good feedback. Had a chat with Han, and commented on this section. This is not useful to a user. We used to send data to multiple clusters. :)

Reply:
Commented out just incase someone requests to do this in the future. Then, we already have the content :)